trade-finance-manager-ui/server/controllers/utilisation-reports/index.ts

```diff
@@ -22,10 +21,6 @@ export const getUtilisationReports = async (req: Request, res: Response) => {
       isPDCReconcileUser: isPDCReconcileUser(user),
     };
 
-    if (!isTfmPaymentReconciliationFeatureFlagEnabled()) {
```

Comment:
I think there is a chance there are some fields on the view model that are not needed anymore e.g. download links, which we might then also be able to remove from the api response. Would you mind double-checking?